worlds/faxanadu/__init__.py

```diff
@@ -160,19 +165,13 @@ def create_items(self) -> None:
                 for i in range(item.progression_count):
                     itempool.append(FaxanaduItem(item.name, ItemClassification.progression, item.id, self.player))
 
-        # Set up filler ratios
-        self.filler_ratios = {
-            item.name: item.count
-            for item in Items.items
-            if item.classification in [ItemClassification.filler, ItemClassification.trap]
-        }
-
+        # Adjust filler ratios
         # If red potions are locked in shops, remove the count from the ratio.
         self.filler_ratios["Red Potion"] -= red_potion_in_shop_count
 
         # Remove poisons if not desired
-        if not self.options.include_poisons:
-            self.filler_ratios["Poison"] = 0
+        if self.options.include_poisons:
+            self.filler_ratios["Poison"] = 13
```

Comment:
We could grab the number `13` from the ItemDef or add a constant for it in Items.py
Not a big deal anyway. Just that number is in 2 different places.

Reply:
Should be addressed now